lib/lotus/repository.rb

```diff
@@ -674,18 +674,18 @@ def _persisted?(entity)
 
       # Add time create an entity
       def _update_created_at(entity)
-        if entity.class.attributes.include?(:created_at)
+        if entity.respond_to?(:created_at) && entity.respond_to?(:created_at=)
           entity.created_at ||= Time.now.utc
         end
       end
 
       # Add time update an entity
       def _update_updated_at(entity)
-        if entity.class.attributes.include?(:updated_at)
-          if entity.updated_at
-            entity.updated_at = Time.now.utc
+        if entity.respond_to?(:updated_at) && entity.respond_to?(:updated_at=)
+          if entity.respond_to?(:created_at) && entity.created_at
+            entity.updated_at ? entity.updated_at = Time.now.utc : entity.updated_at = entity.created_at
```

Comment:
@AlfonsoUceda: As you see method update timestamps
```
 def _update_timestamps(entity)
    _update_created_at(entity)
    _update_updated_at(entity)
 end
```

`created_at && updated_at` will get different `time.now.utc` value. Moreover, In the first time, when you create an entity, you don't touch `updated_at` thus `updated_at` can't accept a different value with `created_at`. Hence, the value of `updated_at` must equal `created_at`.